package.json

```diff
@@ -46,6 +46,7 @@
     "eslint-plugin-react": "^7.1.0",
     "extract-text-webpack-plugin": "3.0.0",
     "file-loader": "0.11.2",
+    "firebase": "^4.3.0",
```

Comment:
This should be `"firebase-tools": "^3.9.2",`

The `firebase` package doesn't include the cli tool.

Reply:
You're absolutely right. Fixed.